api/src/wfl/api/spec.clj

```diff
@@ -106,12 +114,12 @@
 (s/def ::identifier string?)
 (s/def ::fromOutputs map?)
 (s/def ::fromSource string?)
-(s/def ::labels (s/* string?))
+(s/def ::labels (s/* util/label?))
 (s/def ::name string?)
 (s/def ::methodConfiguration (s/and string? util/terra-namespaced-name?))
 (s/def ::methodConfigurationVersion integer?)
 (s/def ::table string?)
-(s/def ::watchers (s/* string?))
+(s/def ::watchers (s/* email-address?))
 (s/def ::workspace (s/and string? util/terra-namespaced-name?))
 (s/def ::snapshots (s/* ::uuid))
```

Comment:
It might be better to accept a JSON map for labels and then compose what the external services need.

Reply:
I decided against this so multiple labels with the same name can be given:
```["project:Test Boston" "project:COVID-19 Surveillance" ...]```

Reply:
Cool. I did not think of that.